src/pages/journal/components/candidate-link/candidate-link.ts

```diff
@@ -22,10 +21,14 @@ export class CandidateLinkComponent {
   @Input()
   isPortrait: boolean;
 
-  constructor(public navController: NavController) {
+  constructor(public modalController: ModalController) {
   }
 
-  navigateToCandidateDetails() {
-    this.navController.push('CandidateDetailsPage', { slotId: this.slotId, slotChanged: this.slotChanged });
+  openCandidateDetailsModal() {
```

Comment:
quick test around this?

Reply:
there is a test in there for the openCandidateDetailsModal